libknet/compress_model.h

```diff
@@ -11,7 +11,14 @@
 
 #include "internals.h"
 
-#define KNET_COMPRESS_MODEL_ABI 1
+#define KNET_COMPRESS_MODEL_ABI            1
+#define KNET_COMPRESS_UNKNOWN_DEFAULT    (-1)
+#define LZ4_COMPRESS_DEFAULT               1 /* lz4 default compression level from lz4.c */
+#define ZLIB_COMPRESS_DEFAULT              6 /* zlib default compression level from zlib.h */
```

Comment:
one more question here.. for example looking at zlib.h, I can see default compression is:
Z_DEFAULT_COMPRESSION (-1) that as the header says is 6.
I think it would make sense to use the header #define for this.

For example:
#define ZLIB_COMPRESS_DEFAULT Z_DEFAULT_COMPRESSION

this would require moving those defines back into the modules, but the benefit is that if a library changes default, then we can pick it up at the next build without having to change the hardcoded numbers here.

Make sense?